client/src/pages/Suggest/index.jsx

```diff
@@ -23,10 +25,12 @@ export default function Suggest() {
 				Object.entries(formData).filter(([, value]) => value !== "")
 			);
 			try {
-				await resourceService.suggest(suggestion);
+				const resource = await resourceService.suggest(suggestion);
 				setMessage({
 					success: true,
-					text: "Thank you for suggesting a resource!",
+					text: resource.draft
+						? "Thank you for suggesting a resource!"
+						: "Thank you for publishing a resource!",
```

Comment:
✅ This is neat, basing it on the response means it continues to work even if the logic around roles changes later on.